db/re/skill_db.yml

```diff
@@ -6305,9 +6305,9 @@ Body:
     Type: Weapon
     TargetType: Attack
     DamageFlags:
-      IgnoreFlee: true
+      IgnoreFlee: false
```

Comment:
I would be inclined to remove this flag all together instead of flipping it to false. False is default so the only reason to explicitly define it as false is if you're using import/skill_db. Don't forget to remove the head label as well (`DamageFlags`).